From c36e5c33bcafb070f9e22e1248e34f14d630add4 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Thu, 26 Apr 2012 09:46:49 +0100
Subject: [PATCH 507/507] mageia: Correct usage of M4_DEFINES vs.
 M4_DISTRO_FLAG.

This seems to have been a problem since mageia support was merged,
as upstream had changed how this bit worked without us reaslising.
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8ae5ab2..07da43e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -593,7 +593,7 @@ case $with_distro in
         mageia)
                 SYSTEM_SYSVINIT_PATH=/etc/rc.d/init.d
                 AC_DEFINE(TARGET_MAGEIA, [], [Target is Mageia])
-                M4_DISTRO_FLAG=-DTARGET_MAGEIA=1
+                M4_DEFINES=-DTARGET_MAGEIA=1
                 have_plymouth=yes
                 ;;
         other)
-- 
1.7.10.3

